tests/test_badpixelmap.py

```diff
@@ -0,0 +1,96 @@
+import os
+import glob
+import numpy as np
+import corgidrp.data as data
+import corgidrp.mocks as mocks
+import corgidrp.detector as detector
+from corgidrp.bad_pixel_calibration import create_bad_pixel_map
+
+
+def test_badpixelmap(): 
+    '''
+
+    Tests the creation of badpixelmaps.
+
+    Create master darks and master flats, inject some hot and cold pixels
+    to create a master bad pixel map. 
+
+    '''
+
+    ###### create simulated dark data
+    # check that simulated data folder exists, and create if not
+    datadir = os.path.join(os.path.dirname(__file__), "simdata")
+    if not os.path.exists(datadir):
+        os.mkdir(datadir)
+    
+    mocks.create_dark_calib_files(filedir = datadir)
+
+    ####### test data architecture
+    dark_filenames = glob.glob(os.path.join(datadir, "simcal_dark*.fits"))
+
+    dark_dataset = data.Dataset(dark_filenames)
+
+    ###### create dark
+    dark_frame = detector.create_dark_calib(dark_dataset)
```

Comment:
note that this is being removed in PR #105 in favor of `darks.build_trad_dark`. Is that ok here?

Reply:
ok, moved to using `build_trad_dark`